src/containers/collectibles/components/CollectiblesPage.tsx

```diff
@@ -534,6 +655,58 @@ const CollectiblesPage: React.FC<{
         </div>
       </Modal>
 
+      <Modal
+        isOpen={isEmbedModalOpen}
+        onClose={closeEmbedModal}
+        showDismissButton
+        showTitleHeader
+        title={collectibleMessages.embedCollectibles}
+        bodyClassName={styles.embedModalBody}
+        titleClassName={styles.embedModalTitle}
+        headerContainerClassName={styles.embedModalHeader}
+        allowScroll
+        zIndex={zIndex.COLLECTIBLE_EMBED_VIEW_MODAL}
+      >
+        <div className={styles.embedContainer}>
+          <div
+            className={styles.embedFrame}
+            dangerouslySetInnerHTML={{ __html: embedIFrameUrl }}
```

Comment:
can we not reuse this component? https://github.com/AudiusProject/audius-client/blob/be456a65c14b2a2c6e85b734c2b258f0f122eb37/src/containers/embed-modal/components/EmbedFrame.tsx#L16

Reply:
good call, not sure why I didn't the first time